braph2genesis/graphs/_MultilayerBU.gen.m

```diff
@@ -0,0 +1,433 @@
+% ¡header!
+MultilayerBU < Graph (g, multilayer binary undirected graph) is a multilayer binary undirected graph.
+
+%%% ¡description!
+In a multilayer binary undirected (BU) graph, layers have same or different number 
+ of nodes with within-layer undirected edges either 0 (absence of connection) 
+ or 1 (existence of connection).
+The connectivity matrices are symmetric.
+There are connections between layers connecting the corresponding nodes.
+
+%% ¡props_update!
+
+%%% ¡prop!
+
+NAME (constant, string) is the name of the multilayer binary undirected graph.
+%%%% ¡default!
+'MultilayerBU'
+
+%%% ¡prop!
+DESCRIPTION (constant, string) is the description of the multilayer binary undirected graph.
+%%%% ¡default!
+'In a multilayer binary undirected (BU) graph, layers have same or different nodes with within-layer undirected edges either 0 (absence of connection) or 1 (existence of connection). The connectivity matrices are symmetric. There are connections between layers connecting the corresponding nodes.'
```

Comment:
In a multilayer binary undirected (BU) graph, layers could have different number of nodes with within-layer undirected edges, associated with either 0 (absence of connection) or 1 (existence of connection). The connectivity matrices are symmetric (within layer). All node connections are allowed between layers.